### DISCUSSION TITLE
Nextjs+prisma+mysql+cpanel not working

### DISCUSSION THREAD

**Discussion by mahbubhswe** [prisma/prisma#16475] [Q&A]
i am try to access mysql database from local...it work without prisma....when i run npx prisma migrate dev....give me this error
![Screenshot_4](https://user-images.githubusercontent.com/99654070/204144480-350b5efc-315f-4e67-bd92-26795fd5f1f1.jpg)

**Comment by nurul3101** [comment]
Hello @mahbubhswe 👋 

This [section](https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database#shadow-database-user-permissions) in our docs shows what permissions are required by shadow database for each database that we support.
Can you have a look at it and grant these permissions? That should fix the issue for you.

Let me know if you still run into this issue after granting permissions.

**Reply by vladi-stevanovic** [reply]
👋 Hi @mahbubhswe , did you perhaps have a chance to check Nurul's recommendation? Let us know if you still have any questions! 

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
